<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XmlAddressParseClass</class>
 <widget class="QMainWindow" name="XmlAddressParseClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XmlAddressParse</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="mainTab">
       <attribute name="title">
        <string>Main</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item row="0" column="0">
         <widget class="QScrollArea" name="dataScroll">
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>298</width>
             <height>532</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLineEdit" name="findHousesEdit">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="findHousesButton">
                <property name="text">
                 <string>Find Houses</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <widget class="QTextBrowser" name="resultsBrowser">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QScrollArea" name="mapScroll">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>630</width>
             <height>532</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_4"/>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="otherTab">
       <attribute name="title">
        <string>Other</string>
       </attribute>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>70</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Find Additional</string>
        </property>
       </widget>
       <widget class="QPushButton" name="openButton">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>30</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="openEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>401</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>check_diff.xml</string>
        </property>
       </widget>
       <widget class="QLabel" name="openlabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>401</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Set xml file</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="cityBox">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>130</y>
          <width>129</width>
          <height>82</height>
         </rect>
        </property>
        <property name="title">
         <string>City</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <property name="verticalSpacing">
          <number>20</number>
         </property>
         <item row="0" column="0">
          <widget class="QRadioButton" name="mskButton">
           <property name="text">
            <string>Msk</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="spbButton">
           <property name="text">
            <string>Spb</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="areaBox">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>230</y>
          <width>129</width>
          <height>82</height>
         </rect>
        </property>
        <property name="title">
         <string>Area</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <property name="verticalSpacing">
          <number>20</number>
         </property>
         <item row="0" column="0">
          <widget class="QRadioButton" name="cityButton">
           <property name="text">
            <string>City</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="districtButton">
           <property name="text">
            <string>District</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
